ci/cloudbuild/builds/publish-docs.sh

```diff
@@ -73,9 +73,22 @@ if [[ "${PROJECT_ID:-}" != "cloud-cpp-testing-resources" ]]; then
   exit 0
 fi
 
+# See b/242562806 for the motivation. To update ci/docuploader-requirements.txt you need to do something like:
```

Comment:
Does "b/..." have any currency in OSS land?

Reply:
I hope it is clearer now.